src/cascadia/TerminalControl/TermControl.idl

```diff
@@ -105,6 +106,10 @@ namespace Microsoft.Terminal.TerminalControl
         void SendInput(String input);
         void ToggleRetroEffect();
 
+        void TaskbarProgressChanged();
+        UInt64 GetTaskbarState();
+        UInt64 GetTaskbarProgress();

```

Comment:
should these be property getters? We've been trying to avoid `GetXxx` methods in preference for `UInt64 Xxx { get; };`

I know changing this will diff all the new code you've written, so perhaps it's a NIT